lib/pretty_qr_painter.dart

```diff
@@ -21,7 +21,7 @@ class PrettyQrCodePainter extends CustomPainter {
       this.typeNumber = 1,
       this.image}) {
     _qrCode = QrCode(typeNumber, errorCorrectLevel);
-    _qrCode.addData(data);
+    _qrCode.addData(data!);
```

Comment:
QrCode object cannot be null right?
and I made sure that the data is not null by the bang operator.

Reply:
Hi @SasiZets! 👋

I think it's not good to cast `data`, using the `!` operator. Instead, it would be good to do:

```dart
class PrettyQrCodePainter extends CustomPainter {
  final String? data; // -> final String data;

  PrettyQrCodePainter({...}) {
    _qrCode.addData(data!); // -> _qrCode.addData(data); ;
  }
}
```

Reply:
I didn't cast anything bro, I just migrated the project to thanks safety and did v2 migration.